index.html

```diff
@@ -4,9 +4,9 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Welcome to potionfy</title>
 </head>
 <body>
-    
+    <h1>Welcome to potionfy! Enjoy your stay</h1>
```

Comment:
Could you remove 'enjoy your stay' string and make the 'Welcome to potionfy' red?